leda/Setup.md

```diff
@@ -18,69 +18,83 @@ limitations under the License.
 
 SPDX-License-Identifier: Apache-2.0
 -->
-# Setting Up a Leda Image
+The components of the Fleet Management blueprint fall into two categories: the *kuksa.val Databroker*,
+*CSV Provider* and *FMS Forwarder* components are supposed to run in the vehicle, whereas the remaining
+components are supposed to run in a (cloud) back end to which the vehicle is connected via (public internet)
+networking infrastructure.
 
-The setup for this FMS scenario assumes a split into components running in the vehicles and components
-running in a (cloud) backend. The following page describes how to configure a Leda instance to become part
-of this FMS setup. More precisely, we deploy the containers for the feedercan, the fms-forwarder,
-and the Kuksa.val databroker with an FMS-specific vehicle model.
+All of the components can be run on a single Docker host as described in the [top level README](../README.md). However, the following sections describe a more realistic deployment scenario in which
+[Eclipse Leda](https://eclipse-leda.github.io/leda/) is used as the vehicle runtime environment.
 
-## Run Leda
-The following steps assume a running instance of Eclipse Leda. For more details see the
-[Eclipse Leda Getting Started](https://eclipse-leda.github.io/leda/docs/general-usage/)
-This guide was tested with release v0.1.0-M1 of Eclipse Leda.
+This guide was tested with release v0.1.0-M2 of Eclipse Leda.
 
-## Backup existing manifestes
-In Leda, there are manifests to manage the execution of containers by Eclipse Kanto. During the next steps we
-will overwrite some of the default manifests (`databroker.json`, `feedercan.json`). Because of that,
-we recommend to backup the existing manifests from `/data/var/containers/mainfests`.
+# Start Back End Components
 
-## Copy manifests to Leda
-To trigger the execution of the required containers for the FMS setup, copy the manifest files from
-`leda/data/var/containers/manifests` in the host to `/data/var/containers/manifests` in Leda:
+The containers for the back end components are run on the local host using Docker Compose:
 
-```
-manifests % scp -P 2222 *.json root@127.0.0.1:/data/var/containers/manifests
+```sh
+# in this repository's root folder
+docker compose -f fms-blueprint-compose.yaml up influxdb grafana fms-server --detach
 ```
 
-## Create folders in Leda mounted in mainfests
-The containers described within the manifests require files that are not present in Leda like the FMS-specific vehicle model. 
-Therefore, we need to create the paths (e.g,. `mkdir`) in Leda from which the containers try to mount these files which are:
+# Start In-Vehicle Coponents
 
-- `mkdir -p /data/usr/fms/dbc`
-- `mkdir -p /data/usr/fms/databroker`
+The containers for the in-vehicle components are deployed to a Leda instance running on the
+local host. The *FMS Forwarder* running in Leda will then connect to the *influxdb* server running
```

Comment:
I think it good that we explicitly mention that this guide assumes the in-vehicle (Leda) and the back end side on the same host. What do you think about adding a hint that they could in general run on different hosts (Pi and VM in the Cloud)?

Reply:
FMPOV this should be covered as part of adding a description (in another PR) of how to set it up using Eclipse Hono. In a cluod based setup I would not suggest to connect the vehicle(s) directly to influxdb ...

Reply:
Ok, this could be part of another PR as well. I would not be that strict about only using Hono in a cloud based setup. If we still talk about an example deployment or have a setup where the "Cloud" runs in the local network, e.g., on developer computer and the in-vehicle stack on a RaspberryPi, going for deploying Hono adds overhead for the computation, deployment of Hono. 
To me the hint of running it in a local host, may raise the question for a newcomer what would break if you do not use a localhost setup while technically it is not required (even there might be reasons to go for the Hono based setup).

Reply:
Any suggestion as to what wording we should include?

Reply:
Yes, I created a PR in #8